src/Services/Filters.js

```diff
@@ -29,10 +31,20 @@ const filters = createSlice({
     updateRecsListFilters(state, action) {
       state.recsListState = action.payload;
     },
+    sortTableIndex(state, action) {
```

Comment:
Setters or "update actions" should have an appropriate name in the style "[**update**|**set**]SortIndex" or similar.